mixins/localize/README.md

```diff
@@ -127,59 +127,60 @@ render() {
 
 ## `localizeHTML()`
 
-You can include some basic markup in your localization resources and safely convert them to HTML with the `localizeHTML()` method.
+Rich formatting can be included in localization resources and safely converted to HTML with the `localizeHTML()` method.
 
-### Simple Markup
+### Basic Formatting
 
-#### Bold
+The following formatting elements are supported out-of-the-box:
 
-The `[b]bold[/b]` markup converts to the `<strong>` tag.
+* `<p>paragraphs</p>`
+* `line<br></br>breaks` (note the end tag is required)
+* `<b>bold</b>`
+* `<strong>strong</strong>`
+* `<i>italic</i>`
+* `<em>emphasis</em>`
 
-#### Italic
+Remember that `<strong>` is for content of greater importance (browsers show this visually using bold), while `<b>` only bolds the text visually without increasing its importance.
 
-The `[i]italic[/i]` markup converts to the `<em>` tag.
+Similarly `<em>` *emphasizes* a particular piece of text (browsers show this visually using italics), whereas `<i>` only italicizes the text visually without emphasis.
```

Comment:
Currently our handling makes `<b>` and `<strong>` result in the exact same `<strong>` HTML (and likewise with i/em). Is that fine, or do you want them to actually keep their meaning?

Reply:
I think they should keep their meaning to avoid confusion. I've updated the replacement code to do this.